docs/conf.py

```diff
@@ -117,7 +117,10 @@
 # region Options for extensions
 
 intersphinx_mapping = {
-    "craft-parts": ("https://canonical-craft-parts.readthedocs-hosted.com/en/latest/", None),
+    "craft-parts": (
+        "https://canonical-craft-parts.readthedocs-hosted.com/en/latest/",
+        None,
+    ),
```

Comment:
If we merge https://github.com/canonical/snapcraft/pull/5605 first, we'll have to drop these lines.